.eslintrc.json

```diff
@@ -17,6 +17,10 @@
     ],
     "rules": {
         "@typescript-eslint/no-var-requires": 1,
-        "no-prototype-builtins": 0
-    }
+        "no-prototype-builtins": 0,
+        "@typescript-eslint/no-explicit-any": 0
+    },
+    "ignorePatterns": [
+        "**/jabra-*/*.ts"
```

Comment:
I have this here since we are gonna try and move away from jabra-chrome and jabra-native.  I suppose for the time being however, I should really just ignore jabra-chrome since that was replaced with just the newer implementation